js/ui/button_group.js

```diff
@@ -268,7 +269,9 @@ const ButtonGroup = Widget.inherit({
             selectionRequired: false,
             onItemRendered: e => {
                 const width = this.option("width");
+                const height = this.option("height");
                 isDefined(width) && $(e.itemElement).addClass(BUTTON_GROUP_ITEM_HAS_WIDTH);
+                isDefined(height) && $(e.itemElement).addClass(BUTTON_GROUP_ITEM_HAS_HEIGHT);
```

Comment:
Add separate test for the ButtonGroup widget

Reply:
Got it, thanks

Reply:
Got it, thanks